src/util-napatech.c

```diff
@@ -978,9 +986,14 @@ uint32_t NapatechSetupTraffic(uint32_t first_stream, uint32_t last_stream,
 
             char copystr[16];
             strlcpy(copystr, port->val, sizeof(copystr));
-
-            start = atoi(copystr);
-            end = atoi(strchr(copystr, '-') + 1);
+            if (ByteExtractStringUint8(&start, 10, 0, (const char *)copystr) < 0) {
```

Comment:
Different pattern than what is added around line 522. I like this better. It's slightly more verbose but more readable.